webkit2/webkit2.uri-scheme-request.lisp

```diff
@@ -0,0 +1,105 @@
+;;; webkit2.uri-scheme-request.lisp --- bindings for WebKitURISchemeRequest
+
+;; This file is part of cl-webkit.
+;;
+;; cl-webkit is free software; you can redistribute it and/or modify
+;; it under the terms of the MIT license.
+;; See `COPYING' in the source distribution for details.
+
+;;; Code:
+
+(in-package #:webkit2)
+
+(define-webkit-class "WebKitURISchemeRequest" () ())
+
+(define-g-object-class "GInputStream" g-input-stream
+    (:export nil
+     :type-initializer "g_input_stream_get_type")
+    ())
+(define-g-object-class "GMemoryInputStream" g-memory-input-stream
+    (:superclass g-input-stream
+     :export nil
+     :type-initializer "g_memory_input_stream_get_type")
+    ())
```

Comment:
I guess this belongs to cl-cffi-gtk, right?

If so I suggest moving these two to a separate file and add a comment explaining why we had to add them, and that ideally we should upstream them in the future.